compileComponents.js

```diff
@@ -4,11 +4,30 @@ const bundleMapping = require('./componentsList');
 const path = require('path');
 const fs = require('fs');
 
+const component_array = [];
```

Comment:
lets revert this file and i'll merge this in and release it

Reply:
Will do that in a few minutes